scripts/checksum

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env node
+
+var shell = require('shelljs');
+var helpers = require('./helpers');
+var crypto = require('crypto');
+
+function checksum(filename, assignment, rootChecksum) {
+  if(assignment) {
+    var assignmentConfig = shell.cat('exercises/' + assignment + '/' + filename).toString();
```

Comment:
Can you use `[path, element].join('/')` or a string template instead? It's quite easy to read this incorrectly, and difficult to see where the boundaries are.

In this case I prefer: `['exercises', assignment, filename].join('/')`